### PR TITLE
Renamed timestamp_t to duration_t

### PR DESCRIPTION
**Pull request by jbrazio**
As suggested by Scott, "duration" is a better term for this structure.

### CODE REVIEW

**Comment by petrzjunior** [comment]
In M31:

``` cpp
SERIAL_ECHO_START;
SERIAL_ECHOPGM(MSG_PRINT_TIME " ");
SERIAL_ECHOLN(buffer);
```

Are you sure you want to translate "Print time"? I've always seen Serial messages as untranslated.

**Comment by jbrazio** [comment]
Good point, I would say no.
This must be changed before merging.

File: Marlin/Marlin_main.cpp [hunk 1]
```diff
@@ -4064,7 +4064,7 @@ inline void gcode_M31() {
   lcd_setstatus(buffer);
 
   SERIAL_ECHO_START;
-  SERIAL_ECHOPGM(MSG_PRINT_TIME " ");
+  SERIAL_ECHOPGM("Print time: ");
```
Comment (petrzjunior) [inline Marlin/Marlin_main.cpp:4067]: So `MSG_PRINT_TIME` isn't used anymore, right? Should be removed from language files ASAP (#4395)

**Comment by petrzjunior** [comment]
This one is merged, but I can still see `MSG_PRINT_TIME` in language files (en, it, kana, kana-utf8). Are there any plans to use it or should I skip translating it? Then someone (namely @jbrazio :smile:) could remove it from these files.

**Pull request merged**